server/routes/prisons/sessionTemplates/editSessionTemplateController.ts

```diff
@@ -91,7 +91,7 @@ export default class EditSessionTemplateController {
       req.flash('formValues', formValues)
       return res.render('pages/prisons/sessionTemplates/editSingleSessionTemplate', {
         errors: req.flash('errors'),
-        message: req.flash('message'),
+        messages: req.flash('messages'),
         formValues,
         // incentiveGroups,
```

Comment:
Do these comments need to be here? Or at least another comment explaining why they're commented out?

Reply:
Not the first time you've spotted this, @psoleckimoj - see https://github.com/ministryofjustice/hmpps-visits-internal-admin-ui/pull/87#discussion_r1288204986!

That was 2 years ago so I think it's safe to say they can go. Have pushed up a change to remove them.